.rubocop_todo.yml

```diff
@@ -0,0 +1,33 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2019-10-08 20:34:01 +0300 using RuboCop version 0.75.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Performance/RegexpMatch:
+  Exclude:
+    - 'lib/yavdb/sources/snyk_io.rb'
+
+# Offense count: 1
+# Configuration parameters: Max.
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/crawler_spec.rb'
+
+# Offense count: 1
+# Configuration parameters: CustomTransform, IgnoreMethods.
+RSpec/FilePath:
```

Comment:
This one seems harder. I am open to suggestions. I am not an experienced Ruby developer and this project has a lot of experimentation. Since I come from Scala/Java some patterns might be weird here and I am completely open to suggestions.
Do you have any? We can also leave it ignored for now.

Reply:
idea of this cop is to check folder structure and it's recommended to have folder structure as in app/gem itself
so path of `snyk` spec here should be `spec/yavdb/sources/snyk_io_spec.rb`

Reply:
Sounds good. Can you change it in this PR?

Reply:
sure 👍

Reply:
another problem with this is that class is called `Client`, so cop recommends to use `spec/yavdb/sources/snyk_io/client_spec.rb`, not sure if it's clear and easy to find spec with such filename in IDE